releasenotes/notes/fix-jinja-env-81c98225b22dc827.yaml

```diff
@@ -0,0 +1,11 @@
+---
+security:
+  - |
+    Fix issue that could lead to remote code execution when using insecure Jinja template in the following Components:
+
+    - `PromptBuilder`
+    - `ChatPromptBuilder`
+    - `OutputAdapter`
+    - `ConditionalRouter`
+
+    The same issue has been fixed in the `PipelineTemplate` class too.
```

Comment:
We should mention that this fix will disable support for user types as outputs of templates (as a breaking change).